ci-operator/step-registry/rosa/conf/idp/htpasswd/rosa-conf-idp-htpasswd-commands.sh

```diff
@@ -123,3 +123,15 @@ done
 # Store kubeconfig
 echo "Kubeconfig file: ${KUBECONFIG}"
 cat ${KUBECONFIG} > "${SHARED_DIR}/kubeconfig"
+
+# Check if runtime_env exists and then check if $USERS is not empty
+user="${IDP_USER}:${IDP_PASSWD}"
+# store users in a shared file
+if [ -f "${SHARED_DIR}/runtime_env" ] ; then
+  source "${SHARED_DIR}/runtime_env"
+fi
+runtime_env="${SHARED_DIR}/runtime_env"
+
+cat <<EOF >>"${runtime_env}"
+export USERS=${user}
+EOF
```

Comment:
We generated the multiple user/password in https://github.com/openshift/release/blob/master/ci-operator/step-registry/osd-ccs/conf/idp/htpasswd/multi-users/osd-ccs-conf-idp-htpasswd-multi-users-commands.sh#L36-L54. It is a history that in the past, rosa CLI does not support configure multiple users so we used ocm CLI. 

If the current rosa CLI does support multiple users, let us do this in the step `rosa-conf-idp-htpasswd-commands` to replace the step `osd-ccs-conf-idp-htpasswd-multi-users`.